pyntc/devices/ios_device.py

```diff
@@ -241,7 +241,8 @@ def boot_options(self):
 
         match = re.search(boot_path_regex, show_boot_out, re.MULTILINE)
         if match:
-            boot_path = match.group(1)
+            boot_path_tuple = match.groups()
+            boot_path = [value for value in boot_path_tuple if value is not None][0]
```

Comment:
I like the comprehension! Is this always going to be the first not None? Maybe you can exit earlier if it is the first element and not form a whole list?

Reply:
Add in a comment to describe what you are doing there?

Reply:
@mundruid the list comp will always return a list of one value